database/migrations/2020_02_28_192503_add_module_id_to_suggested_resources_table.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddModuleIdToSuggestedResourcesTable extends Migration
+{
+    public function up()
+    {
+        Schema::table('suggested_resources', function (Blueprint $table) {
+            $table->unsignedBigInteger('module_id')->nullable();
+            $table->foreign('module_id')->references('id')->on('modules');
+        });
+    }
+
+    public function down()
+    {
+        Schema::table('suggested_resources', function (Blueprint $table) {
+            $table->dropForeign('module_id');
```

Comment:
I know it's strange, but the `dropForeign` syntax wants the fully generated key, not the name you passed `foreign`.

So, that would be, I think, `suggested_resources_module_id_foreign`, although you should check out whether that works correctly.